Dockerfile

```
@@ -0,0 +1,17 @@
+FROM debian:bullseye
+MAINTAINER Nikolay Korotkiy <nikolay.korotkiy@wirenboard.com>
+
+ENV CORE_PATH=/Z-Uno-G2-Core
+ENV ARM_GCC_PATH=/gcc
+ENV ARM_GCC_VERSION=7.2.1
+ENV LIBCLANG_PATH=/libclang
+ENV PATH="${PATH}:/zme_make"
+
+RUN apt update && apt install -y make unzip wget && \
```

```suggestion
RUN apt-get update && apt-get install -y make unzip wget && \
```

`apt` издевается над консолью ради красивого прогрессбара, `apt-get` лучше использовать в скриптах